templates/flightfact.cmds.erb

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+<% cmd_base="flight fact set #{@asset_data.name}" %>
```

Comment:
It's probably a good idea to quote the asset name.  I'm not sure how common spaces are, but we can support them easily enough.

```
<% cmd_base="flight fact set '#{@asset_data.name}'" %>
```